RadixWallet/Features/DebugInspectProfile/DebugInspectProfile.swift

```diff
@@ -16,11 +16,8 @@ public struct DebugInspectProfile: Sendable, FeatureReducer {
 		}
 
 		public var json: String? {
-			@Dependency(\.jsonEncoder) var jsonEncoder
-			let encoder = jsonEncoder()
-			encoder.outputFormatting = [.prettyPrinted, .withoutEscapingSlashes, .sortedKeys]
 			guard
-				let json = try? encoder.encode(profile),
+				case let json = profile.profileSnapshot(),
```

Comment:
Maybe it's just me but I hate this usage of `case`. Also, what does "snapshot" mean here? And, how is it different from .jsonData?

Reply:
"snapshot" is something we have used to not that "this is a snapshot in time of the every changing organism that is Profile". It is now somewhat of a legacy term, but used in many places in the code base. in the coming weeks/months more and more of code touching it will move into Sargon, so will be refactored eventually.

Big picture now Gustaf... :D is this really blocking merge?

Reply:
If you look closely, you will notice that I have not requested any changes. Since I am serializing and deserializing profiles, it is relevant for me to understand the difference between .jsonData or .profileSnapshot, even if it's temporary.

Reply:
@kugel3 the method `jsonData()` calls `profileSnapshot()` so they are the same.